src/test/resources/fixtures/hmh-ri/caliperEventAssessmentStarted.json

```diff
@@ -0,0 +1,58 @@
+{
+  "@context":"http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "type":"AssessmentEvent",
+  "id":"urn:uuid:uniqueIdPerEvent",
+  "actor":{
+    "id":"https://https://www.hmhco.com/ed/users/userRefId",
+    "type":"Person",
+    "name" : "Casandra Rath"
+  },
+  "action":"Started",
+  "object":{
+    "id":"https://www.hmhco.com/readinginventory/quiz/c35603f7a3434e2cb74ef14d60e70f42/instance/belonging/to/student/userRefId",
+    "type":"Assessment",
+    "name":"HMH-RI Quiz Name - Reading Inventory",
+    "dateToStartOn":"2016-11-14T05:00:00.000Z",
+    "dateToSubmit":"2016-11-18T11:59:59.000Z",
+    "maxAttempts":1,
+    "maxSubmits":1,
+    "maxScore":25.0,
+    "version":"1.0"
+  },
+  "generated":{
+    "id":"https://www.hmhco.com/readinginventory/quiz/c35603f7a3434e2cb74ef14d60e70f42/instance/belonging/to/student/userRefId/attempts/1",
+    "type":"Attempt",
+    "assignee":"https://https://www.hmhco.com/ed/users/userRefId", (same as actors id)
+    "assignable":"https://www.hmhco.com/readinginventory/quiz/c35603f7a3434e2cb74ef14d60e70f42/instance/belonging/to/student/userRefId", (same as object id)
+    "count":1,
+    "dateCreated":"2016-11-15T10:15:00.000Z",
+    "startedAtTime":"2016-11-15T10:15:00.000Z" (current date+time)
+  },
+  "eventTime":"2016-11-15T10:15:00.000Z",
+  "edApp":{
+    "id":"https://www.hmhco.com/readinginventory",
+    "type":"SoftwareApplication",
+    "version":"v1"
+  },
+  "group": {
+    "id": "https://url/to/ids/district/school/sections/sectionId1",
+    "type": "CourseSection",
+    "courseNumber": "CPS 435-01",
+    "academicSession": "Fall 2016"
+  },
+  "membership":{
+    "id":"https://url/to/ids/roster/2018/id1", (not sure what id to put here)
+    "type":"Membership",
+    "organization":{
+      "id":"https://url/to/ids/schools/schoolRefId",
+      "type":"Organization",
+      "subOrganizationOf":{
+        "id":"https://url/to/ids/districts/districtRefId",
+        "type":"Organization"
+      }
+    },
+    "roles":[ "Learner" ],
+    "status": "Active",
+    "dateCreated": "2016-08-01T06:00:00.000Z"
+  }
+}
```

Comment:
Should have a session which reflects the test session, and a federatedSession, using a federated session id which comes from the LTI launch (as previously discussed on Slack)

Reply:
The membership object should be the same as for other Caliper events we already handle

Reply:
Thanks @padraig-ohiceadha if "SessionEvent" comes on board it can be added in to "session". 
Same for "federatedSession", if its a available, I'd imagine it can be added here also